less/v2/forms.less

```diff
@@ -420,6 +421,11 @@
     &:hover a { text-shadow: 0 0 6px var(--color-shadow-primary); }
   }
 
+  :is(document-tags, multi-select):has(select:disabled) .tags.input-element-tags .tag{
```

Comment:
In testing this I noticed that you can still delete damage types on weapons when the sheet is disabled (you can even click to delete them on items in locked compendiums, but the change isn't saved of course). I think we fixed that issue for activities inside our custom `_disableFields` implementation which specifically disables `document-tags` and `multi-select` elements.